cypress/tests/data/60-content/VkarbasizaedSubmission.spec.js

```diff
@@ -53,10 +57,151 @@ describe('Data suite tests', function() {
 		cy.get('button').contains('Continue').click();
 		cy.get('button').contains('Continue').click();
 		cy.get('button').contains('Complete').click();
+	});
+
+	it('Schedule for publication', function() {
+		cy.login('dbarnes');
+		// schedule for the publication in the future issue
+		cy.visit('index.php/publicknowledge/submissions');
+		cy.get('button[id="active-button"]').click();
+		cy.contains('View ' + familyName).click({force: true});
+		cy.get('button[id="publication-button"]').click();
+		cy.get('div#publication button:contains("Schedule For Publication")').click();
+		cy.get('select[id="assignToIssue-issueId-control"]').select(issueTitle);
+		cy.get('div[id^="assign-"] button:contains("Save")').click();
+		cy.get('div:contains("All publication requirements have been met. This will be published when ' + issueTitle + ' is published. Are you sure you want to schedule this for publication?")');
+		cy.get('div.pkpWorkflow__publishModal button:contains("Schedule For Publication")').click();
+		// check status = 5 (scheduled)
+		cy.reload() // to be able to get the header
+		// check submission status
+		cy.get('span').should('have.class', 'pkpWorkflow__identificationStatus').contains("Scheduled");
+		// check publication status
+		cy.get('button[id="publication-button"]').click();
+		cy.get('span').should('have.class', 'pkpPublication__status').contains("Scheduled");
+		// the button "Unschedule" exists
+		// the buttons "Create New Version" (connected with submission) and "Unpublish" (connected with publication) does not exist
+		cy.get('div#publication button:contains("Unschedule")');
+		cy.get('div#publication button:contains("Create New Version")').should('not.exist');
+		cy.get('div#publication button:contains("Unpublish")').should('not.exist');
+
+		// isInTOC:
+		cy.visit('index.php/publicknowledge/manageIssues#future');
+		cy.get('a:contains("' + issueTitle + '")').click();
+		cy.get('div[id^="component-grid-toc-tocgrid-"] span:contains("' + submissionTitle + '")');
+		// go to issues management page to go around closing the TOC window
+		cy.visit('index.php/publicknowledge/manageIssues');
```

Comment:
This line shouldn't be necessary. If the previous line passes, the test is passed so the visit can be dropped to save a bit of time.

Reply:
Here the TOC window stays open until one clicks on the "x". That's why I am visiting the page anew. Eventually I can check if I can close that window here, but I believe I took a look into it and didn't find a solution... I can double check...

Reply:
Why do you need to close the TOC modal? At the end of a Cypress test the session is cleared completely.

Reply:
oh, I see, this is the last line in a test function :-) :+1: